html/includes/common/availability-map.inc.php

```diff
@@ -256,8 +256,14 @@
     }
 
     if (($mode == 1 || $mode == 2) && ($config['show_services'] != 0)) {
-        $service_query = 'select `S`.`service_type`, `S`.`service_id`, `S`.`service_desc`, `S`.`service_status`, `D`.`hostname`, `D`.`sysName`, `D`.`device_id`, `D`.`os`, `D`.`icon` from services S, devices D where `S`.`device_id` = `D`.`device_id` ORDER BY '.$serviceOrderBy.';';
-        $services = dbFetchRows($service_query);
+        if ($_SESSION['userlevel'] >= '5') {
```

Comment:
You can change this to `if (is_normal_user() === false) {` rather than using sessions direct.

Reply:
Neat, thanks!